> Compilation failure needs to be fixed:
> 
> ```
> [ERROR] /home/jenkins/workspace/Core_jenkins_PR-3873/core/src/main/java/hudson/model/AbstractCIBase.java:[96,5] error: cannot find symbol
> 
> [ERROR]   symbol:   class Restricted
> 
> [ERROR]   location: class AbstractCIBase
> ```
> IIRC there is also a second usage of the class in the AdministrativeMonitor

Yes, there is a second usage in public void `disable` class. And in that I have done changes.
`Set<String> set = jenkins.getDisabledAdministrativeMonitors();` 
 instead of  `Set<String> set = jenkins.disabledAdministrativeMonitors();`
But same compilation error continues.

--- a/core/src/main/java/hudson/model/AbstractCIBase.java
+++ b/core/src/main/java/hudson/model/AbstractCIBase.java
@@ -88,11 +88,6 @@ public abstract class AbstractCIBase extends Node implements ItemGroup<TopLevelI
         c.kill();
     }
 
-    /* =================================================================================================================
-    * Package-protected, but accessed API
-    * ============================================================================================================== */
-
-    /*package*/
     @Restricted(NoExternalUse.class)
     public CopyOnWriteArraySet<String> getDisabledAdministrativeMonitors(){
     	return new CopyOnWriteArraySet<String>();

--- a/core/src/main/java/hudson/model/AdministrativeMonitor.java
+++ b/core/src/main/java/hudson/model/AdministrativeMonitor.java
@@ -117,7 +117,7 @@ public abstract class AdministrativeMonitor extends AbstractModelObject implemen
      */
     public void disable(boolean value) throws IOException {
         AbstractCIBase jenkins = Jenkins.get();
-        Set<String> set = jenkins.disabledAdministrativeMonitors;
+        Set<String> set = jenkins.getDisabledAdministrativeMonitors();
         if(value)   set.add(id);
         else        set.remove(id);
         jenkins.save();
@@ -131,7 +131,7 @@ public abstract class AdministrativeMonitor extends AbstractModelObject implemen
      * he wants to ignore.
      */
     public boolean isEnabled() {
-        return !(Jenkins.get()).getDisabledAdministrativeMonitors().contains(id);
+        return !Jenkins.get().getDisabledAdministrativeMonitors().contains(id);
     }
 
     /**

